packages/core/src/Machine.ts

```diff
@@ -11,6 +11,15 @@ import {
 import { StateNode } from './StateNode';
 import { Model, ModelContextFrom, ModelEventsFrom } from './model';
 
+/**
+ * Docs: {@link https://xstate.js.org/docs/guides/machines.html#configuration Machines}.
+ * API: {@link https://xstate.js.org/api/globals.html#machine Machine}.
```

Comment:
I wonder if this can be automated somehow? 🤔 (just thinking for the future)

Reply:
That would be super awesome!! No idea how though 😂

Reply:
@davidkpiano As in like something that would automatically add the links to jsdoc blocks?

Reply:
Could you write a script that extracts a map the `id` of a headers/sub headers to its url from the documentation, then parses through the source code looking for jsdoc blocks that contain that  an certain`id` and injects the mapped link into the jsdoc?

Reply:
I guess more generically you need a script the extract a concept and its link from documentation, then another script that finds jsdoc blocks and determines what concept it best matches to and injects that concepts link into the jsdoc block!